From 1591558a82332ec467f5b6548f751d159eb30a8e Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Mon, 12 Jun 2023 15:32:11 +0200
Subject: [PATCH 0337/1141] arm64: dts: st: rename stm32mp257f-ev1 memory
 mapping

fixup! arm64: dts: st: introduce stm32mp257f-ev memory mapping

Official board name is stm32mp257f-ev1.

Change-Id: I08d086ece1afa61b3d8044830d3d3b1e383be7e0
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/312288
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
---
 ...5tdcid-resmem.dtsi => stm32mp257f-ev1-ca35tdcid-resmem.dtsi} | 0
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts                      | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename arch/arm64/boot/dts/st/{stm32mp257f-ev-ca35tdcid-resmem.dtsi => stm32mp257f-ev1-ca35tdcid-resmem.dtsi} (100%)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev-ca35tdcid-resmem.dtsi b/arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi
similarity index 100%
rename from arch/arm64/boot/dts/st/stm32mp257f-ev-ca35tdcid-resmem.dtsi
rename to arch/arm64/boot/dts/st/stm32mp257f-ev1-ca35tdcid-resmem.dtsi
diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 93e2196154e6..09a175a097f4 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -14,7 +14,7 @@
 #include "stm32mp25xf.dtsi"
 #include "stm32mp25-pinctrl.dtsi"
 #include "stm32mp25xxai-pinctrl.dtsi"
-#include "stm32mp257f-ev-ca35tdcid-resmem.dtsi"
+#include "stm32mp257f-ev1-ca35tdcid-resmem.dtsi"
 
 / {
 	model = "STMicroelectronics STM32MP257F-EV1 Evaluation Board";
-- 
2.39.2

